src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -38,14 +40,14 @@
   private final boolean overflow;
   private final String territoryName;
   private final IUIContext uiContext;
-  private final static List<String> STATIC_UNITS =
-      Arrays.asList(Constants.UNIT_TYPE_AAGUN, Constants.UNIT_TYPE_FACTORY);// TODO add every "static"
-  // non-movable unit type of every
-  // map.
   private static UnitFlagDrawMode drawUnitNationMode = UnitFlagDrawMode.NEXT_TO;
+  
+  public static final String UNIT_FLAG_DRAW_MODE = "UNIT_FLAG_DRAW_MODE";
```

Comment:
This string is used for storing the preferences... Since I need the property more than once, I added the variable here, so that the name could easily be changed